### PR TITLE
Koa2

### PR DESCRIPTION
**Pull request by omsmith**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,42 +38,42 @@ module.exports = function (options) {
     : typeof options.threshold === 'string' ? bytes(options.threshold)
     : 1024
 
-  return function* compress(next) {
-    this.vary('Accept-Encoding')
-
-    yield* next
+  return (ctx, next) => {
```
Comment (omsmith) [inline index.js]: Was following style in koajs/koa, I think, but might opt to not make this one an arrow in order to maintain the name - for stack purposes

Comment (coderhaoxin) [inline index.js]: I'd rather keeping the function name

**Comment by fundon** [comment]
Ping @coderhaoxin

**Comment by niftylettuce** [comment]
+1

**Comment by ifraixedes** [comment]
This cannot be merged due some conflicts.

Any plan to rebase it and any of the @contributors available to review it?

**Comment by jonathanong** [comment]
let me try to merge this. i bet it's some stupid conflict like `traivs.yml`

**Comment by omsmith** [comment]
I can rebase it - just give me a few minutes

**Comment by jonathanong** [comment]
awesome :+1:

**Comment by omsmith** [comment]
@jonathanong should be good.

Haven't actually "looked at it" again, but fixed the package.json conflict.

**Comment by jonathanong** [comment]
published as v2.x and tagged as `next`

**Pull request closed without merging**